src/org/opendatakit/briefcase/ui/export/ExportForms.java

```diff
@@ -3,28 +3,39 @@
 import static java.util.stream.Collectors.toList;
 import static java.util.stream.Collectors.toMap;
 
-import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 import org.opendatakit.briefcase.model.BriefcaseFormDefinition;
+import org.opendatakit.briefcase.model.BriefcasePreferences;
 import org.opendatakit.briefcase.model.FormStatus;
 
 public class ExportForms {
   private final List<FormStatus> forms;
-  private final Map<FormStatus, ExportConfiguration> configurations = new HashMap<>();
+  private final Map<FormStatus, ExportConfiguration> configurations;
   private Map<String, FormStatus> formsIndex = new HashMap<>();
 
-  ExportForms() {
-    forms = new ArrayList<>();
-  }
-
-  public ExportForms(List<FormStatus> forms) {
+  public ExportForms(List<FormStatus> forms, Map<FormStatus, ExportConfiguration> configurations) {
     this.forms = forms;
+    this.configurations = configurations;
     rebuildIndex();
   }
 
+  public static ExportForms load(List<FormStatus> forms, BriefcasePreferences preferences) {
+    // This should be a simple Map filtering block but we'll have to wait for Vavr.io
+
+    Map<FormStatus, ExportConfiguration> configurations = new HashMap<>();
+    forms.forEach(form -> {
+      ExportConfiguration configuration = ExportConfiguration.load(preferences, "custom_" + form.getFormName() + "_");
```

Comment:
I hadn't considered before now that Briefcase uses form name a lot (directory naming, etc). That's potentially dangerous as there's no uniqueness guarantee but I suppose there isn't one for form_id across multiple servers, either. Using form name here is the consistent way to go but thought I'd mention this just in case we run into issues related to it at some point.

Reply:
Maybe we should use UUID values. I haven't reviewed which method Aggregate uses to generate its UIDs but they are quite like UUIDs... Those could be used safely across multiple servers.